packages/esm-ward-app/src/hooks/useInpatientRequest.ts

```diff
@@ -1,15 +1,22 @@
 import { type FetchResponse, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
-import type { InpatientRequest } from '../types';
+import type { InpatientRequestFetchResponse } from '../types';
 import useSWR from 'swr';
 
 export function useInpatientRequest(locationUuid: string) {
-  const { data, ...rest } = useSWR<FetchResponse<Array<InpatientRequest>>, Error>(
-    locationUuid ? `${restBaseUrl}/emrapi/inpatient/admissionRequests?admissionLocation=${locationUuid}` : null,
+  // prettier-ignore
+  const customRepresentation =
+    'custom:(dispositionType,' +
+      'patient:(uuid,identifiers,voided,' +
+        'person:(uuid,display,gender,age,birthdate,birthtime,preferredName,preferredAddress,dead,deathDate,voided)))'
+  const { data, ...rest } = useSWR<FetchResponse<InpatientRequestFetchResponse>, Error>(
+    locationUuid
+      ? `${restBaseUrl}/emrapi/inpatient/request?dispositionType=ADMIT,TRANSFER,DISCHARGE&dispositionLocation=${locationUuid}&v=${customRepresentation}`
```

Comment:
This was a mistake/confusion in my example, but we *don't* want to retrieve DISCHARGE requests here